analyzers.yml

```
@@ -153,3 +153,7 @@ analyzers:
     name: TyScan
     github: sider/TyScan
     doc: tools/javascript/tyscan
+  pylint:
+    name: pylint
```

It seems to me that the official name is **Pylint** instead of pylint.

```suggestion
    name: Pylint
```